### ISSUE TITLE
Adjust "headings" styles

### ISSUE THREAD

**Issue by frankiegranato** [canonical-websites/insights.ubuntu.com#240]

**Comment by frankiegranato** [comment]
Can we give authors an option to choose less 'shouty' sub headings (ie. smaller type size)? 
Also, can we clip the line length to match the line length of the paragraphs? (this might make the text look tidier - especially when there are lots of sub headings in the article as there are in this example:  https://insights.ubuntu.com/2018/01/31/plex-joins-snapcraft-summit-to-advance-snap-learnings

See screen shot: ![Ubuntu Insights.png](https://images.zenhubusercontent.com/593552bf51aa2c3057806d8c/c5ae3f58-a9b0-4541-9639-1459838004db)

**Comment by steverydz** [comment]
We can't change the headings font-size in the template. It looks like they've been added in WordPress. Editors should be able to change the headings to h3 if required.

The long headings issue was caused by the paragraphs having a max-width set. I've wrapped the whole post in a container with the same max-width so headings will wrap in the same place:

https://github.com/canonical-websites/insights.ubuntu.com/pull/249
